cgi/CRMS.pm

```diff
@@ -8169,7 +8171,14 @@ sub BibRights {
   my $self = shift;
 
   use BibRights;
-  return BibRights->new;
+  BibRights->new;
+}
+
+sub Field008Formatter {
+  my $self = shift;
+
+  use CRMS::Field008Formatter;
+  CRMS::Field008Formatter->new;
 }
 
 1;
```

Comment:
If we can keep this class below 10K lines that would be great.

Reply:
Should be following a shrinking trend as long as the cleanup tickets keep making it into sprints.